tools/RAiDER/models/gmao.py

```diff
@@ -0,0 +1,171 @@
+import datetime
+import numpy as np
+import os
+from pyproj import CRS
+import re
+
+from RAiDER.models.weatherModel import WeatherModel
+
+
+def Model():
+    return GMAO()
+
+class GMAO(WeatherModel):
+    # I took this from GMAO model level weblink
+    # https://opendap.nccs.nasa.gov/dods/GEOS-5/fp/0.25_deg/assim/inst3_3d_asm_Nv
+    def __init__(self):
+        # initialize a weather model
+        WeatherModel.__init__(self)
+
+        self._humidityType = 'q'
+        self._model_level_type = 'ml' # Default, pressure levels are 'pl'
+        
+        self._classname = 'gmao'
+        self._dataset = 'gmao'
+
+        # Tuple of min/max years where data is available. 
+        self._valid_range = (datetime.datetime(2017,12,1),"Present")
+        self._lag_time = datetime.timedelta(hours=0.3125) # Availability lag time in days
```

Comment:
Comment says "lag time in days" but "hours" keyword argument is used. Is this correct?

Reply:
So is the lag time is really supposed to be 18.75 minutes? Then I would use the minutes argument so it's more clear. If it's supposed to be 0.3125 days (7.5 hours) then I would use `hours=7.5`.